.github/workflows/hint.yml

```
@@ -0,0 +1,33 @@
+name: Hint to a github contributor
+
+on:
+  pull_request_target:
+    types: [opened, created]
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest

```

```suggestion
    runs-on: ubuntu-latest
    permissions:
      contents: write
```

https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#create-an-issue
> The fine-grained token must have the following permission set:
>
> "Issues" repository permissions (write)

Default permissions for repositories depend on when the repository/user/organization/enterprise was created. Newer organizations are created with more restrictive permissions. Older entities should be upgraded to restricted permissions, but until they are, you'll be able to do things w/o declaring permissions and will experience problems trying to share those workflows to restricted containers.

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-the-default-github_token-permissions